src/main/java/arsenic/utils/rotations/RotationUtils.java

```diff
@@ -1,36 +1,153 @@
 package arsenic.utils.rotations;
 
+import arsenic.main.Arsenic;
 import arsenic.utils.java.JavaUtils;
 import arsenic.utils.java.UtilityClass;
-import arsenic.utils.minecraft.PlayerUtils;
 import net.minecraft.client.Minecraft;
-import net.minecraft.util.BlockPos;
-import net.minecraft.util.EnumFacing;
-import net.minecraft.util.MathHelper;
-import net.minecraft.util.Vec3;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.util.*;
 
 public class RotationUtils extends UtilityClass {
 
     private static final Minecraft mc = Minecraft.getMinecraft();
 
+    public static float[] getRotationsToEntity(EntityLivingBase e, int smooth) {
+        if (e == null) return null;
+
+        double x = e.posX - mc.thePlayer.posX;
+        double y = e.getPositionVector().yCoord - mc.thePlayer.getPositionVector().yCoord;
+        double z = e.posZ - mc.thePlayer.posZ;
+        double distance = MathHelper.sqrt_double((x * x) + (z * z));
+
+        float targetYaw = (float) ((Math.toDegrees(Math.atan2(z, x))) - 90);
+        float targetPitch = (float) (-Math.toDegrees(Math.atan2(y, distance)));
+
+        float random1 = (float) JavaUtils.getRandom(-1, 1);
+        float random2 = (float) JavaUtils.getRandom(-1, 1);
+
+        if (random1 == random2) {
+            while (random1 == random2) {
```

Comment:
why use a while loop? just change random1's value to smth lese manually instead of using random when both randoms are equal

Reply:
that 0.00001 chance its the same again